src/components/WebexRemoteMedia/WebexRemoteMedia.jsx

```diff
@@ -22,9 +25,11 @@ export default function WebexRemoteMedia({className, meetingID}) {
     remoteShare,
     error,
   } = useMeeting(meetingID);
+  const members = useMemberships(meetingID, 'meeting');
   const audioRef = useStream(remoteAudio);
   const videoRef = useStream(remoteVideo);
   const shareRef = useStream(remoteShare);
+  const hasMembers = members.length > 1;
```

Comment:
`hasMembers` by itself is a little misleading, as "self" is a current member. We want to know if there are others besides self.

```suggestion
  const hasOtherMembers = members.length > 1;
```